native/machine-coding/Movie-Info-App/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="./index.js"></script>
+    <script src="../../helpers/header.js" type="module"></script>
+    <link rel="stylesheet" href="./index.css">
+    <title>Movie Info Website</title>
+</head>
+<body>
+    <header>
+        <h1>Movie Info Website</h1>
+    </header>
```

Comment:
This part of the html is not required because a header will already be created by the `helpers/header.js` script.

Also remove the corresponding CSS.